File: Practice38/1.py
class c:
    def __init__(self, v):
        self.value = v
    def getValue(self):
        print(self.value)
    def setValue(self, v):
        self.value = v

c1 = c(10)
c1.getValue()
c1.setValue(20)
c1.getValue()

File: Practice38/1.rb
class C
    def initialize(v)
        @value = v
    end
    def show()
        p @value
    end
    def getValue()
        return @value
    end
    def setValue(v)
        @value = v
    end
end

c1 = C.new(10)

p c1.getValue()
p c1.setValue(20)
c1.show()